app/vmui/packages/vmui/src/components/Main/TextField/style.scss

```diff
@@ -150,17 +150,17 @@
 
   &__icon-start,
   &__icon-end {
-    align-self: center;
+    align-content: center;
```

Comment:
This breaks the styles, for example on the Cardinality page:
<img width="405" height="88" alt="Image" src="https://github.com/user-attachments/assets/a1571476-3ef3-40d6-bf81-bb24895c4bc8" />